src/emulator/io/src/io.cpp

```diff
@@ -117,6 +117,7 @@ bool init(IOState &io, const char *pref_path) {
     fs::create_directory(ux0_savedata);
     fs::create_directory(uma0);
     fs::create_directory(uma0_data);
+    fs::create_directory(shaders_path);
```

Comment:
This is not really an IO thing, it's more of a GXM thing. I saw we create the title-specific subdirectory in `dump_missing_shader`. We should probably attempt to create the `shaderlog` directory here too.

Reply:
yes, i have say same, shaderlog folder is better for me, and same, inside same bin folder

Reply:
@Zangetsu38 I already asked him to rename it to `shaderlog`, it's done (never mind the variable name).
Also, yes, they'll be both placed in bin folder. `shaders` for pre-existing shaders, and `shaderlog` for generated ones. So I guess @GusAndrianos you'll have to make `load_shader` also look in "/shaders" instead of just `shaders_path` ("shaderlog").

Reply:
@VelocityRa oh ok, just i have also say that first time in discord ^^